CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ The format is based on [Keep a Changelog].
 
 - `IBMQJob` now has three new methods: `done()`, `running()`, and
   `cancelled()`. The methods are used to indicate the job status. (\#494)
+- `IBMQBackend` now has a new `jobs_limit()` method that returns the 
+  jobs limit for a backend, such as the current number of running jobs 
+  on the backend and also the maximum number of jobs that could be 
+  submitted to the backend. (\#513)
```

Comment:
Can you check with the api team to see if the limit is per provider? And clarify that here and the method's docstring.

Reply:
From what I have seen, the limits for a user are for a given provider/device combination. But, I'll double check with them 👍